tests/general/LineLengthBearTest.py

```diff
@@ -26,3 +34,50 @@
     settings={
         'max_line_length': '4',
         'ignore_length_regex': 'http://, https://, ftp://'})
+
+
+LineLengthBearLangSpecificLineLengthTest = verify_local_bear(
+    LineLengthBear,
+    valid_files=(test_file,),
+    invalid_files=(invalid_VB_file,),
+    settings={'language': 'VisualBasic'},
+        )
+
+
+LineLengthBearValidLanguageTest = verify_local_bear(
+    LineLengthBear,
+    valid_files=(test_file,),
+    invalid_files=(invalid_general_file,),
+    settings={'language': 'C'},
+        )
```

Comment:
closing brackets for these `verify_local_bear` calls should be hard set against the left margin